dist/stat_data.py

```diff
@@ -233,7 +233,6 @@ def __init__(self, name, desc, flags=''):
     LogStat('log_prealloc_max',
         'number of pre-allocated log files to create', 'no_clear,no_scale'),
     LogStat('log_prealloc_used', 'pre-allocated log files used'),
-    LogStat('log_reads', 'log read operations'),
     LogStat('log_release_write_lsn', 'log release advances write LSN'),
     LogStat('log_scan_records', 'records processed by log scan'),
     LogStat('log_scan_rereads', 'log scan records requiring two reads'),
```

Comment:
@sueloverso, @michaelcahill: I can't imagine anyone is using the `log_reads` statistic, but we have a request from Amazon to never change existing statistics. Michael?

If we're really going to agree to never change statistics, we should probably put some process in place to guarantee that is the case from release-to-release.